### ISSUE TITLE
Error:Error fetching component from backend

### ISSUE THREAD

**Issue by wattaken** [eclipse/sw360#986]
I try to build install SW360 10.0 in vm ubuntu18
I reffered to this blog  to install.
https://qiita.com/K-Hama/items/cd1c7f6e9e812bb8faf8#sw360%E5%88%9D%E6%9C%9F%E3%83%A6%E3%83%BC%E3%82%B6%E8%A8%AD%E5%AE%9A (git checkout 1b93c88  #sw360-10.0.0-M1)

Everythings looks fine. but import SPDX licenses  is  failed.
I found ERROR in  tomcat catrina.out. 

<img src="https://user-images.githubusercontent.com/43559387/93448383-81289400-f90e-11ea-9e74-44bc3c23447c.png" width="200">

```
2020-09-17 08:26:43.933 ERROR [http-nio-8080-exec-19][PortletServlet:119] javax.portlet.PortletException: org.apache.thrift.transport.TTransportException: HTTP Response code: 500
javax.portlet.PortletException: org.apache.thrift.transport.TTransportException: HTTP Response code: 500
```
this machine in no proxy enviroyment.
fossology is also launch another vm ubuntu.  connection is private_network in virtualbox. 
and both vm have another brige-network NIC.   internet connection is fine.

I can't think of what to do next to solve it.
Please give me some advices.

 i 'm sorry  pushed enter key before input description.

**Comment by KoukiHama** [comment]
Hi, thank you for reading my blog.This issue was solved recently. If you install latest sw360, you can install spdx data.

**Comment by wattaken** [comment]
thank you  
I tried it.   but  I  could not  finish to install 11.0.0 today.

**Comment by wattaken** [comment]
I complete to import SPDX licenses. thank you.
I finished to install the latest release version 11.0.0.

I comment out setenv.sh the follow line.
    #JAVA_OPTS="$JAVA_OPTS -XX:+UseParNewGC"

Is this option  able to on OpenJDK11  ?

**Comment by wattaken** [comment]
more question.
I run startup.sh this error output tomcat log.

catalina.out
```
2020-09-24 09:53:00.295 INFO  [http-nio-8080-exec-3][BuildInfoTemplateContextContributor:60] Build Info Context Contributor: attribute gitBranch=UNKNOWN

2020-09-24 09:53:47 ERROR ProcessFunction:47 - Internal error processing getSubscribedComponents
org.ektorp.DbAccessException: com.fasterxml.jackson.core.JsonParseException: Current token (VALUE_NULL) not numeric, can not use numeric value accessors
 at [Source: org.ektorp.http.StdHttpResponse$ConnectionReleasingInputStream@26b622e4; line: 1, column: 32]
 .....
Caused by: com.fasterxml.jackson.core.JsonParseException: Current token (VALUE_NULL) not numeric, can not use numeric value accessors
 at [Source: org.ektorp.http.StdHttpResponse$ConnectionReleasingInputStream@26b622e4; line: 1, column: 32]
 .....

2020-09-24 09:53:47 ERROR MySubscriptionsPortlet:64 - Could not fetch your subscriptions from backend
org.apache.thrift.TApplicationException: Internal error processing getSubscribedComponents
        at org.apache.thrift.TServiceClient.receiveBase(TServiceClient.java:79) ~[bundleFile:?]
 .....
2020-09-24 09:53:49 ERROR ProcessFunction:47 - Internal error processing getRecentReleases
org.ektorp.DbAccessException: com.fasterxml.jackson.core.JsonParseException: Current token (VALUE_NULL) not numeric, can not use numeric value accessors
 at [Source: org.ektorp.http.StdHttpResponse$ConnectionReleasingInputStream@135dc2ef; line: 1, column: 32]
Caused by: com.fasterxml.jackson.core.JsonParseException: Current token (VALUE_NULL) not numeric, can not use numeric value accessors
 at [Source: org.ektorp.http.StdHttpResponse$ConnectionReleasingInputStream@135dc2ef; line: 1, column: 32]
 .....
2020-09-24 09:53:49 ERROR RecentReleasesPortlet:59 - Could not fetch recent components from backend
org.apache.thrift.TApplicationException: Internal error processing getRecentReleases
```

It appeared to be working properly.
However, when I pressed the Create Component button,
error was ocured 
"" Error:Error fetching component from backend. ""

My couchdb has this component, 
so this compornent is list in http://localhost/en/group/guest/components.
althought, ckick this the same error is apper.


<img src=https://user-images.githubusercontent.com/43559387/94131903-8307cf80-fe99-11ea-8019-fc82cb72236f.png width=400>

catrina.out
```
2020-09-24 10:18:40 ERROR ProcessFunction:47 - Internal error processing getComponentById
org.ektorp.DbAccessException: com.fasterxml.jackson.core.JsonParseException: Current token (VALUE_NULL) not numeric, can not use numeric value accessors
 at [Source: org.ektorp.http.StdHttpResponse$ConnectionReleasingInputStream@4d6e749a; line: 1, column: 32]
...
        at java.lang.Thread.run(Thread.java:834) [?:?]
2020-09-24 10:18:41.000 ERROR [http-nio-8080-exec-1][detail_jsp:294] Error in useBeans: java.lang.InstantiationException: bean usingProjects not found within scope
```

what is wrong?

**Comment by mcjaeger** [comment]
I am not sure what went wrong, because the error message seem to report a very general error. It looks like the data model / schema of the response is not as expected, thus it could be that some of the migration script were not executed when applying an update.

You could try to rebuilt couchdb views, here is how to do that the hard way in couchdb 1.x:

1. got to `/var/lib/couchdb` there shouldbe database files and design folders for each of these
2. stop tomcat
3. stop couchdb
4. delete the `:sw360db_design`folder, or better: place it some where else
5. start cocuhdb
6. start tomcat
7. either open a couple of windows to trigger view creatin
8. or go into the futon interface and select the view "all" for each of the data objects (or at least the import ones, such as release, components
9: watch status of the couchdb until view jobs are done
10. check in sw360

There might be better ways to rebuild views, but that as a quick help.

**Comment by wattaken** [comment]
I re-installed all.  the reason is my vm-instance is broken by my wrong operation. 
Now, sw360 works fine.

I  found that I delated the database _replicator and _users on couchdb.
(install delete install delete ... , I  mistaken.)
<img src="https://user-images.githubusercontent.com/43559387/94917562-b8409d00-04eb-11eb-917a-a48338db0e6f.png" width="300"> this is the current couchdb.

I think  I did the other operation the same way. 
Is this the cause?

**Comment by drej** [comment]
Recently I got the same error, I was working on a RHEL8 server that could only install couchDB from 2.3.1 by yum. But local.ini file for couchdb:2.3.1 doesn't have a httpd_global_handlers section so I didn't know where to add couchdb-lucene setting for it. ( I tried to add section by myself but it doesn't work.)
Now I launch a couchdb:2.1.2 by container seems the problem has been solved. But I'm really worried about whether am I doing the right thing, or is there anything I should know?
